src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -1150,6 +1152,35 @@ class spell_pal_seal_of_vengeance : public SpellScript
     }
 };
 
+// 1022 - Hand of Protection
+class spell_pal_hand_of_protection : public SpellScript
+{
+    PrepareSpellScript(spell_pal_hand_of_protection);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_PALADIN_HAND_OF_PROTECTION });
+    }
+
```

Comment:
```suggestion
```

https://www.azerothcore.org/wiki/core-scripts#spell-scripts

> Spells used within the SpellScript should always be validated at the top of the script.

no spells are used within the script so no need to validate

Reply:
Corrected.